### ISSUE TITLE
subplot cannot zoom in

### ISSUE THREAD

**Issue by chen112p** [pyvista/pyvista#4020]
### Describe the bug, what's wrong, and what you expected.

Hi when plotting subplots using pyvista 0.38 on Jupyter lab, i can only zoom in and out with my mouse scroller on the first plot [0,0]. I remember old version allows me to zoom in and out the plot where my cursor is at. 

I also found that after running the cell that has the plotter, i have to click on the output window to display the plot. If i wait too long kernel will die from it.. 

Junting

### Steps to reproduce the bug.

```python
Insert code here
```

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Tue Feb 14 10:00:29 2023 Eastern Standard Time

                OS : Windows
            CPU(s) : 12
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.8 GiB
       Environment : Jupyter
        GPU Vendor : Intel
      GPU Renderer : Intel(R) UHD Graphics
       GPU Version : 4.5.0 - Build 30.0.101.1404

  Python 3.9.16 (main, Jan 11 2023, 16:16:36) [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.38.1
               vtk : 9.2.5
             numpy : 1.24.2
           imageio : 2.25.0
            scooby : 0.7.1
             pooch : v1.6.0
        matplotlib : 3.6.3
           IPython : 8.9.0
          colorcet : 3.0.1
           cmocean : 2.0
        ipyvtklink : 0.2.3
              tqdm : 4.64.1
            meshio : 5.3.4
        jupyterlab : 3.6.1
         pythreejs : 2.4.1
             trame : 2.2.6
      trame_client : 2.5.1
      trame_server : 2.8.1
         trame_vtk : 2.0.17
--------------------------------------------------------------------------------
```

### Screenshots

![mouse_scrolling](https://user-images.githubusercontent.com/39268233/218774846-4770dec8-c3bc-485a-8a59-bf3c5be40b74.gif)
![plotter_hang](https://user-images.githubusercontent.com/39268233/218775287-846d851c-cbe8-4434-9f68-fb51d5366080.gif)

**Comment by banesullivan** [comment]
Ah I can reproduce this quite easitly with:

```py
import pyvista as pv

pl = pv.Plotter(shape=(2,2))
pl.add_mesh(pv.Sphere())
pl.subplot(0,1)
pl.add_mesh(pv.Cone())
pl.subplot(1,0)
pl.add_mesh(pv.Cube())
pl.subplot(1,1)
pl.add_mesh(pv.Cylinder())
pl.show()
```

It appear this may be an issue with [VTK.js `RemoteView`](https://kitware.github.io/vtk-js/api/Rendering_Misc_RemoteView.html) but I'm struggling to track down where exactly the mouse/scroll event is translated to the server render window. @jourdain, any thoughts?

**Comment by jourdain** [comment]
https://github.com/Kitware/VTK/blob/master/Web/Python/vtkmodules/web/protocols.py#L155-L174

**Comment by psavery** [comment]
A PR in VTK.js was just merged in kitware/vtk-js#2764 that is needed to fix this.

We also need to finish migrating the protocols from VTK to trame-vtk in https://github.com/Kitware/trame-vtk/pull/31

Once that is done, I have a fix for this, where the server is able to figure out which renderer the mouse is over and update the right camera accordingly.

**Comment by psavery** [comment]
This should now be fixed if you update your `trame-vtk` to version 2.4.0.

Note that you may also need to clear your browser cache as well to see the fix, since there were some javascript changes also.
